src/ExpressionEvaluator/Core/Source/ResultProvider/Expansion/MemberExpansion.cs

```diff
@@ -26,7 +26,8 @@ internal sealed class MemberExpansion : Expansion
             DkmClrValue value,
             ExpansionFlags flags,
             Predicate<MemberInfo> predicate,
-            ResultProvider resultProvider)
+            ResultProvider resultProvider,
+            bool isProxyType)
```

Comment:
Did an existing test fail without `isProxyType`? If not, please add a test that demonstrates the difference.